src/ru/mintmanga/src/eu/kanade/tachiyomi/extension/ru/mintmanga/MintManga.kt

```diff
@@ -20,6 +21,16 @@ class MintManga : GroupLe("MintManga", "https://1.mintmanga.com", "ru") {
 
     override val baseUrl by lazy { getPrefBaseUrl() }
 
+    override fun getChapterSearchParams(document: Document): String {
+        val html = document.html()
+
+        val userHashRegex = "user_hash.+'(.+)'".toRegex()
```

Comment:
Where is `user_hash` stored? in a `<script>`? You can use Jsoup for this, so it's narrowed down a bit:
```kt
val scripts = document.select("script").joinToString("\n") { it.data() }
// or
val script = document.selectFirst("script:containsData(user_hash)")
```

or a combination if `user_hash` is referenced in multiple places.